### PR TITLE
feature mobilenetv3

### PR DESCRIPTION
**Pull request by VladislavPatrushev**

### CODE REVIEW

File: src/models/backbones/mobilenetv3.py [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+from src.constructor import BACKBONES
+
+
+def _cfg(url: str = '', **kwargs):
+    return {
+        'url': url,
+        'input_size': (3, 224, 224),
+        'pool_size': (7, 7),
+        'crop_pct': 0.875,
+        'interpolation': 'bilinear',
```
Comment (PososikTeam) [inline src/models/backbones/mobilenetv3.py:28]: Why we need this parameter?

File: src/models/backbones/mobilenetv3.py [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+        'mean': IMAGENET_DEFAULT_MEAN,
+        'std': IMAGENET_DEFAULT_STD,
+        **kwargs
+    }
+
+
+default_cfgs = {
+    'mobilenet_v3_large': _cfg(url=''),
```
Comment (PososikTeam) [inline src/models/backbones/mobilenetv3.py:36]: Can we get weights?

File: src/models/backbones/mobilenetv3.py [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+@BACKBONES.register_class
+def mobilenet_v3_large(pretrained: bool = False, **kwargs):
+    """It's constructing a mobilenet_v3_large model."""
+    return create_mobilenetv3('mobilenet_v3_large', pretrained, **kwargs)
+
+
+@BACKBONES.register_class
+def mobilenet_v3_small(pretrained: bool = False, **kwargs):
+    """It's constructing a mobilenet_v3_small model."""
+    return create_mobilenetv3('mobilenet_v3_small', pretrained, **kwargs)
```
Comment (PososikTeam) [inline src/models/backbones/mobilenetv3.py:234]: spce line

File: src/models/modules/bricks/activations.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class HSwish(nn.Module):
```
Comment (PososikTeam) [inline src/models/modules/bricks/activations.py:5]: Please use https://pytorch.org/docs/stable/generated/torch.nn.Hardswish.html

File: src/models/backbones/mobilenetv3.py (hunk 1 above)
Comment (Animatory) [reply src/models/backbones/mobilenetv3.py]: Just reference

File: src/models/backbones/mobilenetv3.py (hunk 2 above)
Comment (Animatory) [reply src/models/backbones/mobilenetv3.py]: No, TIMM did not train this model

**Comment by Animatory** [comment]
Close due to alternate implementation

**Pull request closed without merging**